dolly-backend-app/src/main/java/no/nav/dolly/mapper/BestillingTpsfStatusMapper.java

```diff
@@ -68,7 +68,7 @@ private static void checkNUpdateStatus(Map<String, Map<String, Set<String>>> err
                 } else {
                     errorEnvIdents.get(status).put(environ, new HashSet<>(Set.of(ident)));
                 }
-            } else {
+            } else if (nonNull(ident)) {
```

Comment:
med bruker_id nullable så ble ident tidvis null under henting av data.. Det høres jo ut som en alvorlig feil, men kan det være noe data i test DB som er såpass ustabilt at det skjer?

Reply:
dette er ok! 👍

Reply:
Det finnes mange bestillinger med bruker_id = null, fordi bruker_id (user_id het det før og inneholdt z-ident) ble innført nettopp for å kunne spore hvem som bestilte hva, og dette er antagelig et år siden eller så, godt ute i dolly sin eksistens.